registration/tests.py

```diff
@@ -0,0 +1,122 @@
+from django.contrib.auth import get_user_model
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.test import TestCase
+from participant.models import Troop
+
+import os
+from io import StringIO
+
+
+class ImportUsersTestCase(TestCase):
+    def tearDown(self):
+        try:
+            os.remove('test.csv')
+        except OSError:
+            pass
+
+    def test_successful_import(self):
+        csv = 'troop_number;troop_name;firstname;lastname;email\n' \
+              '130000;Diözesanverband München-Freising;Peter;Pan;peter.pan@dpsg1300.de\n' \
+              '131700;Bezirk Ebersberg;Tom;Sawyer;tom.sawyer@pfadfinder-ebersberg.de\n' \
+              '131700;Bezirk Ebersberg;Huckleberry;Finn;huckleberry.finn@pfadfinder-ebersberg.de\n'
+
+        with open('test.csv', 'w') as test_csv:
+            test_csv.write(csv)
```

Comment:
Instead of having the content of the csv in-memroy, writing it to disk, to read it again, it would be better to add a method `import.Command.import_lines(lines)` which would be used for every test.
```python
        lines = [['130000','Diözesanverband München-Freising','Peter;Pan;peter.pan@dpsg1300.de'] ,
              ['131700','Bezirk Ebersberg','Tom;Sawyer','tom.sawyer@pfadfinder-ebersberg.de'],
              ['131700','Bezirk Ebersberg','Huckleberry','Finn','huckleberry.finn@pfadfinder-ebersberg.de']]
```

This would make the tests clearer about their intent (test the content of the lines and not the formatting of the input file - currently a csv, but could be a .xls or .odt)

Another test could explicitely test the formatting of the csv file (and if the file does not exist etc)

Reply:
And a successful test should use a file tracked in git (which can be sent to the Lagerbüro as an example of the file need for the import)